src/components/v5/shared/UserInfoPopover/partials/DomainPermissionList/DomainPermissionList.tsx

```diff
@@ -0,0 +1,120 @@
+}
+
+const displayName = 'v5.UserInfoPopover.partials.DomainPermissionList';
+
+const DomainPermissionList: FC<DomainPermissionListProps> = ({
+  domains,
+  isTopContributorType,
+}) => {
+  const { isMultiSigEnabled } = useEnabledExtensions();
+  const rootDomain = domains.find(({ nativeId }) => nativeId === Id.RootDomain);
+
+  return (
+    <div
+      className={clsx(`flex flex-col pr-6 sm:overflow-hidden sm:pr-3.5`, {
+        'pl-[22px]': isTopContributorType,
+        'pl-[23px]': !isTopContributorType,
+      })}
+    >
+      <TitleLabel
+        text={formatText({
+          id: 'userInfo.teamBreakdown.section',
+        })}
+        className="pb-2"
+      />
+      <ul
+        className={clsx(
+          `${styles.userInfoDomainsWrapper} flex flex-col gap-2 sm:max-h-[216px] sm:overflow-y-auto sm:overflow-x-hidden sm:pr-1.5`,
+          {
+            '-mr-1.5 pr-2': domains.length > 4,
+          },
+        )}
+      >
+        {domains.map(
+          ({
+            domainId,
+            domainColor,
+            domainName,
+            permissions,
+            multiSigPermissions,
+            reputationPercentage,
+            reputationRaw,
+          }) => {
+            const isRootDomain = domainId === rootDomain?.domainId;
+
+            return (
+              <li
+                key={domainId}
+                className="grid h-12 min-h-12 w-full grid-cols-[2fr,1fr] items-center rounded border border-gray-100 px-[11px] font-medium sm:min-w-[302px] sm:max-w-[302px]"
+              >
+                <div className="flex flex-row items-center truncate">
+                  <div
+                    className={`mr-1 flex h-[0.625rem] w-[0.625rem] rounded-full ${getTeamColor(domainColor)}`}
+                  />
+                  <span className="truncate whitespace-nowrap text-sm">
+                    {domainName}
+                  </span>
+                </div>
+                <div className="flex justify-end gap-1">
+                  <PermissionTooltip
+                    userPermissionsInDomain={permissions}
+                    userPermissionsInParentDomain={
+                      rootDomain?.permissions || []
+                    }
+                    isRootDomain={isRootDomain}
+                  />
+                  {isMultiSigEnabled && (
+                    <PermissionTooltip
+                      userPermissionsInDomain={multiSigPermissions}
+                      userPermissionsInParentDomain={
+                        rootDomain?.multiSigPermissions || []
+                      }
+                      isRootDomain={isRootDomain}
+                      isMultiSig
+                    />
+                  )}
+                  <Tooltip
+                    placement="top"
+                    className="flex items-center justify-end text-sm font-normal text-gray-900"
+                    tooltipContent={
+                      <Numeral value={reputationRaw} suffix=" pts" />
```

Comment:
Doesn't have to dealt with here since it can also be addressed during the `master` rebase but just a gentle reminder that this should have the decimals value:
```ts
const { colony } = useColonyContext();

<Numeral
  value={reputationRaw}
  suffix=" pts"
  decimals={getTokenDecimalsWithFallback(colony.nativeToken.decimals)}
/>;
```